### PR TITLE
Add Server.Version and cmd/client test code

### PR DESCRIPTION
**Pull request by haiyanmeng**
This PR implements `Server.Version` and adds `cmd/client/main.go` to test `Server.Version`.

### CODE REVIEW

**Comment by haiyanmeng** [comment]
@mrunalp , PTAL.

File: cmd/client/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	c := pb.NewRuntimeServiceClient(conn)
+
+	// Test RuntimeServiceClient.Version
+	version := "715fec664d75c6b5cb5b12718458621d4b75df37"
+	r, err := c.Version(context.Background(), &pb.VersionRequest{Version: &version})
```
Comment (mrunalp) [inline cmd/client/main.go]: We should accept an argument to invoke this API.
For e.g. ./ociclient --runtimeversion

**Comment by haiyanmeng** [comment]
@mrunalp , PTAL.
I added `github.com/urfave/cli` into `cmd/client/main.go`. Do you think we need to somehow let `godep` track it?

**Comment by mrunalp** [comment]
@hmeng-19 Yes, we should vendor it. Try godep save after you compile it and it should modify the dependencies.

**Comment by haiyanmeng** [comment]
@mrunalp , I am trying to run `godep restore` inside `ocid` to install the vendored packages into my `$GOPATH`. However, `godep` has difficulty to restore k8s. 
Do you have any idea about what is going on here?

```
[hmeng@localhost ocid]$ godep restore
# cd /home/hmeng/go/src/github.com/kubernetes/kubernetes; git checkout 60894b9e4089dd5a854b2c71b6f7d7b1e7a6e9f3
fatal: reference is not a tree: 60894b9e4089dd5a854b2c71b6f7d7b1e7a6e9f3
godep: error downloading dep (github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime): exit status 128
godep: Error downloading some deps. Aborting restore and check.
```

**Comment by mrunalp** [comment]
@hmeng-19 It may be because that checkout came from a PR.

**Comment by haiyanmeng** [comment]
@mrunalp , make sense. Thanks.

**Comment by haiyanmeng** [comment]
@mrunalp , PTAL.
I updated the `vendor` dir by running `godep save ./...`, and squashed the commits into a single one.

**Comment by mrunalp** [comment]
I tried testing and the server crashes:

```
[root@dhcp-16-129 ocid]# ./ocid 
2016/07/13 12:13:52 grpc: Server failed to encode response proto: Marshal called with nil
```

```
[root@dhcp-16-129 ocid]# ./ocidclient --runtimeversion 715fec664d75c6b5cb5b12718458621d4b75df37
2016/07/13 12:13:52 transport: http2Client.notifyError got notified that the client transport was broken EOF.
2016/07/13 12:13:53 grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::1]:49999: getsockopt: connection refused"; Reconnecting to "localhost:49999"
2016/07/13 12:13:55 grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp [::1]:49999: getsockopt: connection refused"; Reconnecting to "localhost:49999"
```

**Comment by haiyanmeng** [comment]
@mrunalp , I think the problem is that `godep save` added `github.com/mrunalp/ocid/server` into `vendor`. So my new changes to the `server` package got hidden.

**Comment by haiyanmeng** [comment]
@mrunalp , I am wondering whether it is really reasonable to track `ocid/server` using `godep`. 

Another problem I found is that if `godep` tracks `ocid/server`, then inside `ocid/vendor/github.com/mrunalp/ocid/server/utils.go`, `runtime.Caller` will set the current file path to be `/home/hmeng/go/src/github.com/hmeng-19/ocid/vendor/github.com/mrunalp/ocid/server/utils.go`. Then I need to run `filepath.Dir` multiple times to find the path of `Godeps.json`.

**Pull request closed without merging**